service/src/main/java/com/commercetools/pspadapter/payone/ServiceFactory.java

```diff
@@ -116,19 +127,53 @@ public static void main(String [] args) throws SchedulerException, MalformedURLE
     }
 
     /**
-     * FIXME return shared instance
      * Creates a new commercetools client instance.
      * @return the client
      */
-    public BlockingSphereClient createCommercetoolsClient() {
-        final SphereClientFactory sphereClientFactory = SphereClientFactory.of();
-        return BlockingSphereClient.of(
-                sphereClientFactory.createClient(
-                        serviceConfig.getCtProjectKey(),
-                        serviceConfig.getCtClientId(),
-                        serviceConfig.getCtClientSecret()),
-                DEFAULT_CTP_CLIENT_TIMEOUT,
-                TimeUnit.SECONDS);
+    synchronized
```

Comment:
i reviewed it and found you are right: since we always need all of these services initialized (except optional `OrderService`, but it is lightweight anyway) it's easier to initialize them directly in the constructor and avoid lazy initialization 

fixed in 4d29a22